BannerlordTwitch/BLTAdoptAHero/Actions/AdoptAHero.cs

```diff
@@ -260,6 +294,19 @@ private static (bool success, string message) ExecuteInternal(string args, strin
                 CharacterDevelopmentCampaignBehaivor.DevelopCharacterStats(newHero);
             }
 
+            if (settings.JoinPlayerClan)
+            {
+                if (Clan.PlayerClan.CompanionLimit < Clan.PlayerClan.Companions.Count)
```

Comment:
Why does CompanionLimit matter when joining clan?

Reply:
Fail from me. I should do this test when the newHero want to join companion, not the clan. We don't want to break the companion limit, except if the general option "break companion limit" is selected.

Reply:
Removed join clan which have no effect on the game. You also have to be a companion or familly member (apparently) to be in a clan.